making-demos.md

```diff
@@ -171,6 +171,27 @@ TODO
 
 TODO
 
+### Prepending the Firefox DevTools intro video
+
+It's a good idea to also prepend the Firefox DevTools intro video in your new video so it looks more official.
+
+You can download this video [on GitHub's LFS storage here](https://github.com/devtools-html/demos/blob/master/assets/bumper%201080.mov).
```

Comment:
is it 'relevant' that it is LFS storage? how do they download it? do they need to use an special LFS tool?